handbook/marketing/livestream.md

```diff
@@ -0,0 +1,117 @@
+# Livestream events (Webinars)
+## Overview
+The following information describes the planning and execution of a Sourcegraph livestream event. 
+
+## Livestream information
+
+All livestream events will require the following details for planning:
+
+- Title
+- Date/Time
+- Short Description
+- Speaker Name
+- Speaker Bio
+- Speaker Headshot
+- Livestram Link from Zoom or GotoWebinar
+- Audience
+
+Sourcegraph uses these web applications for running a livestream event:
+
+- GotoWebinar or Zoom:
```

Comment:
We should settle on one or the other. I personally found GotoWebinar to not be a great experience.

Reply:
As we increase online events, GotoWebinar has the best integration with Hubspot.  With Zoom, a person cannot register for multiple events via Hubspot.  Zoom is okay if we are just planning one online meetup at a time.  Also, we have the option of having many to many meetup.    With more users on Zoom these days, maybe the integration with Hubspot will improve soon.